src/Skoruba.IdentityServer4.Admin.BusinessLogic/Helpers/ComboBoxHelpers.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+using Skoruba.IdentityServer4.Admin.BusinessLogic.Dtos.Common;
+
+namespace Skoruba.IdentityServer4.Admin.BusinessLogic.Helpers
+{
+	public static class ComboBoxHelpers
+	{
+		public static void PopulateValuesToList(string values, List<string> list)
```

Comment:
You should add a comment about what `values` or rename it to `jsonValues` or something, right now for the consumer it's not clear it's a JSON string.

Reply:
Good idea, thank you @TomasHubelbauer 👍

Reply:
Done